sklearn/preprocessing/imputation.py

```diff
@@ -374,3 +385,352 @@ def transform(self, X):
             X[coordinates] = values
 
         return X
+
+
+class MICEImputer(BaseEstimator, TransformerMixin):
+    """Basic implementation of MICE package from R.
+    This version assumes all of the features are Gaussian.
```

Comment:
jump a line between long and short description. You might want to mention that this is the same than in R here.